Meta/Lagom/Libraries/LibGUI/WindowServerConnection.cpp

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (c) 2022, Filiph Sandström <filiph.sandstrom@filfatstudios.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
I'd rather just leave the unused parameters unnamed than add this pragma (or slap a [[maybe_unused]] if it makes more sense in context). Just a bit odd to only be doing this in this file.

Reply:
Yeah; that's a left-over from the initial 3hr prototype I forgot about